src/Maker/MakeCrud.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Doctrine\Bundle\DoctrineBundle\DoctrineBundle;
+use Doctrine\Common\Inflector\Inflector;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Doctrine\DoctrineEntityHelper;
+use Symfony\Bundle\MakerBundle\FileManager;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\GeneratorHelper;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Bundle\TwigBundle\TwigBundle;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Security\Csrf\CsrfTokenManager;
+use Symfony\Component\Validator\Validation;
+
+/**
+ * @author Sadicov Vladimir <sadikoff@gmail.com>
+ */
+final class MakeCrud extends AbstractMaker
+{
+    private $fileManager;
+    private $entityHelper;
+
+    public function __construct(FileManager $fileManager, DoctrineEntityHelper $entityHelper)
+    {
+        $this->fileManager = $fileManager;
+        $this->entityHelper = $entityHelper;
+    }
+
+    public static function getCommandName(): string
+    {
+        return 'make:crud';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureCommand(Command $command, InputConfiguration $inputConfig)
+    {
+        $command
+            ->setDescription('Creates crud for Doctrine entity class')
+            ->addArgument('entity-class', InputArgument::OPTIONAL, sprintf('The class name of the entity to create crud (e.g. <fg=yellow>%s</>)', Str::asClassName(Str::getRandomTerm())))
```

Comment:
We'll need to add the auto-complete for this as well, just like you did for `make:form`.